psst-gui/src/ui/preferences.rs

```diff
@@ -212,6 +212,22 @@ fn general_tab_widget() -> impl Widget<AppState> {
 
     col = col.with_spacer(theme::grid(3.0));
 
+    col = col
+        .with_child(Label::new("Step Duration").with_font(theme::UI_FONT_MEDIUM))
```

Comment:
I think this should be called "Skip Duration" or "Seek Duration"

Reply:
Good call